docs/sources/helm-charts/mimir-distributed/release-notes/v4.5.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Grafana Mimir Helm chart version 4.5 release notes"
+menuTitle: "V4.5 release notes"
+description: "Release notes for Grafana Mimir Helm chart version 4.5"
+weight: 300
+---
+
+# Grafana Mimir Helm chart version 4.5 release notes
+
+Grafana Labs is excited to announce version 4.5 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.9 or Grafana Enterprise Metrics (GEM) v2.9. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+To learn about new features and enhancements in Grafana Mimir or GEM, see the release notes for [Grafana Mimir v2.9](/docs/mimir/v2.9.x/release-notes/v2.9/) or [Grafana Enterprise Metrics v2.9](/docs/enterprise-metrics/v2.9.x/release-notes/v2-9/).
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+> **Note:** If you are upgrading from a previous major version, review the changelog entry for all the major releases since then and perform the necessary migrations.
+
+## Important changes
+
+- **The `graphite-web` component was removed** from the graphite proxy. The `graphite-web` component had several configuration issues which meant it was failing to process requests.
```

Comment:
I'd add more details from the PR that changed it. What was graphite-web responsible for and why was it removed? I think #5133 has a good description, so maybe you can just copy from there


> graphite-web is used to [render queries that the native query engine cannot](https://grafana.com/docs/enterprise-metrics/latest/graphite/graphite_querier/#offloading-the-function-processing-to-graphite).

> the majority of graphite functions and features are supported by the native GEM graphite query engine, and we're currently working on supporting the remaining few issues.

Reply:
thank you, I've formatted the text and added it, going to ask @osg-grafana for a review after the fact. Merging to get the chart tested.